File: src/game/events/game/RoundPassedGameEvent.java
package game.events.game;

import game.Bomberman;
import game.events.models.GameEvent;

/**
 * Fires a game event using Strategy Pattern;
 */
public class RoundPassedGameEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        Bomberman.getMatch().getCurrentLevel().onRoundPassedGameEvent();
    }
}

File: src/game/engine/events/game/RoundPassedGameEvent.kt
package game.engine.events.game

import game.Bomberman
import game.engine.events.models.GameEvent


class RoundPassedGameEvent : GameEvent {
    override fun invoke(arg: Any?) {
        Bomberman.getMatch().currentLevel!!.eventHandler.onRoundPassedGameEvent()
    }
}